airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```
@@ -4773,6 +4917,8 @@ definitions:
         examples:
           - "source_declarative_manifest.components.MyCustomConfigTransformation"
       $parameters:
+        title: "$parameters"
+        description: "Set parameters that are inherited by all child components. Parameters are automatically applied to component fields unless that field already has a value. Parameters propagate recursively to nested components. See https://docs.airbyte.com/platform/connector-development/config-based/advanced-topics/parameters for details."
         type: object
         description: Additional parameters to be passed to the custom config transformation.
```

YAMLlint (1.37.1)

[error] duplication of key "description" in mapping

(key-duplicates)